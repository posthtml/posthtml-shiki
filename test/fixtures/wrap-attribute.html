<shiki lang="js" wrap="div">import { codeToHtml } from 'shiki'</shiki>
