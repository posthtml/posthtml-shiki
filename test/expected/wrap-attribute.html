<div><pre class="shiki nord" style="background-color:#2e3440ff;color:#d8dee9ff" tabindex="0"><code><span class="line"><span style="color:#81A1C1">import</span><span style="color:#ECEFF4"> {</span><span style="color:#8FBCBB"> codeToHtml</span><span style="color:#ECEFF4"> }</span><span style="color:#81A1C1"> from</span><span style="color:#ECEFF4"> '</span><span style="color:#A3BE8C">shiki</span><span style="color:#ECEFF4">'</span></span></code></pre></div>
